muxers/mplex/src/lib.rs

```diff
@@ -85,6 +86,14 @@ impl MplexConfig {
         self
     }
 
+    /// Sets the frame size used when sending data. Capped at 1Mbyte as per the
+    /// Mplex spec.
+    pub fn split_send_size(&mut self, size: usize) -> &mut Self {
+        let size = cmp::min(size, codec::MAX_FRAME_SIZE);
```

Comment:
I pondered that – it's what Yamux does, and I like things to be coherent – but decided for this in the end as it'd be weird to change the return type for this setter only and forego the chainability that was chosen for the other setters. Also: it is not an arbitrary cap but the one from the spec, so if you've chosen to use Mplex, being aware of the specs is expected.

Reply:
I tend to agree on @dvdplm here.
@twittner Any objection to merging?

Reply:
I still do not like the silent capping and would prefer an explicit `Result` but I do not want to be in the way if you prefer otherwise.

Reply:
You can see it a different way: frames are capped twice, once by 1MB because of specs, and once by this limit. The fact that in reality we only cap once is an optimization.